README.md

```diff
@@ -1,3 +1,10 @@
+[![Unit tests](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/workflows/Unit%20tests%20workflow/badge.svg)](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/actions)
+[![Integration tests](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/workflows/E2E%20tests%20workflow/badge.svg)](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/actions)
```

Comment:
Same link for unit and integration tests? Will these two badges show differently if one succeeds and another fails?

Reply:
If I understand correctly, the badge link is tied to the specific workflow, so they will still be independent of each other regarding pass/fail. I think the link is just to go to the general github actions page to see the status of all of the workflows.

Reply:
Yes. Tyler's understanding is correct: badge svg link is tied to the specific workflow. I will change the link from Github action to specific workflow in Github action